### PR TITLE
[Merged by Bors] - feat: add some lemmas about `Finsupp.indicator`

### PR DESCRIPTION
**Pull request by astrainfinita**
mathlib3 PR: https://github.com/leanprover-community/mathlib/pull/17413

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
Did you generate this by hand, or with `mathport`'s one-shot mode? If the former, please don't do this in future; wait for mathport to generate the ported code for you so that we don't need to review it.

File: Mathlib/Data/Finsupp/Defs.lean [hunk 1]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Scott Morrison
 
 ! This file was ported from Lean 3 source module data.finsupp.defs
-! leanprover-community/mathlib commit 8631e2d5ea77f6c13054d9151d82b83069680cb1
+! leanprover-community/mathlib commit 842328d9df7e96fd90fc424e115679c15fb23a71
```
Comment (eric-wieser) [inline Mathlib/Data/Finsupp/Defs.lean:7]: This file [still needs](https://leanprover-community.github.io/mathlib-port-status/file/data/finsupp/defs?range=8631e2d5ea77f6c13054d9151d82b83069680cb1..6623e6af705e97002a9054c1c05a980180276fc1) a forward-port of leanprover-community/mathlib#18277, which would be best as a standalone PR. It's possible there's nothing to port there.

Comment (joneugster) [reply Mathlib/Data/Finsupp/Defs.lean:7]: done in #2653

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

**Comment by eric-wieser** [comment]
bors merge

I think I was just waiting for CI
